From a15eef73fc5a71a3106cc125f5a8f312c6504e89 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 18 Oct 2019 21:42:12 -0700
Subject: [PATCH 36/38] Disable LLDP listening by default.

Do not listen on a raw socket by default for this. Let the
local admin explicitly enable it instead.

https://github.com/clearlinux/distribution/issues/1368
---
 src/network/networkd-network.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/networkd-network.c b/src/network/networkd-network.c
index 39ea4eddd0..ce8cf0b6b7 100644
--- a/src/network/networkd-network.c
+++ b/src/network/networkd-network.c
@@ -451,7 +451,7 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 .priority = LINK_BRIDGE_PORT_PRIORITY_INVALID,
                 .multicast_router = _MULTICAST_ROUTER_INVALID,
 
-                .lldp_mode = LLDP_MODE_ROUTERS_ONLY,
+                .lldp_mode = LLDP_MODE_NO,
                 .lldp_multicast_mode = _SD_LLDP_MULTICAST_MODE_INVALID,
 
                 .dns_default_route = -1,
-- 
2.36.1

